docs/reference/reference-training/training/api-pytorch-reference.rst

```diff
@@ -8,105 +8,112 @@
 
 .. autoclass:: determined.core._distributed.DistributedContext
    :members:
    :inherited-members:
    :member-order: bysource
    :noindex:
 
-***************************************************
- ``determined.pytorch.PyTorchExperimentalContext``
-***************************************************
+``determined.pytorch.PyTorchExperimentalContext``
+=================================================
 
 .. autoclass:: determined.pytorch.PyTorchExperimentalContext
    :members:
    :exclude-members: reduce_metrics, reset_reducers, wrap_reducer
 
 .. _pytorch-dataloader:
 
-***********************************
- ``determined.pytorch.DataLoader``
-***********************************
+``determined.pytorch.DataLoader``
+=================================
 
 .. autoclass:: determined.pytorch.DataLoader
    :members:
 
-************************************
- ``determined.pytorch.LRScheduler``
-************************************
+``determined.pytorch.LRScheduler``
+==================================
 
 .. autoclass:: determined.pytorch.LRScheduler
    :members:
    :special-members: __init__
 
-********************************
- ``determined.pytorch.Reducer``
-********************************
+``determined.pytorch.Reducer``
+==============================
 
 .. autoclass:: determined.pytorch.Reducer
    :members:
 
 .. _pytorch-metric-reducer:
 
-**************************************
- ``determined.pytorch.MetricReducer``
-**************************************
+``determined.pytorch.MetricReducer``
+====================================
 
 .. autoclass:: determined.pytorch.MetricReducer
    :members: reset, per_slot_reduce, cross_slot_reduce
    :member-order: bysource
 
 .. _pytorch-samplers:
 
-*********************************
- ``determined.pytorch.samplers``
-*********************************
+``determined.pytorch.samplers``
+===============================
 
 .. automodule:: determined.pytorch.samplers
    :members:
 
 .. _pytorch-callbacks:
 
-****************************************
- ``determined.pytorch.PyTorchCallback``
-****************************************
+``determined.pytorch.PyTorchCallback``
+======================================
 
 .. autoclass:: determined.pytorch.PyTorchCallback
    :members:
 
 .. _pytorch-writer:
 
-***************************************************************
- ``determined.tensorboard.metric_writers.pytorch.TorchWriter``
-***************************************************************
+``determined.tensorboard.metric_writers.pytorch.TorchWriter``
+=============================================================
 
 .. autoclass:: determined.tensorboard.metric_writers.pytorch.TorchWriter
 
-********************************************************
- ``determined.pytorch.load_trial_from_checkpoint_path``
-********************************************************
+``determined.pytorch.load_trial_from_checkpoint_path``
+======================================================
```

Comment:
somewhere you probably need to update the docs for this function to indicate that it is deprecated, and cannot be used with Trainer-based training loops.